release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -281,6 +281,18 @@ The `openshiftVersion` parameter of the `AgentServiceConfig` custom resource (CR
 
 For more information see, xref:../scalability_and_performance/routing-optimization.adoc#configuring-ingress-controller-router-liveness-readiness-and-startup-probes[Configuring Ingress Controller (router) Liveness, Readiness, and Startup probes]
 
+[id=ocp-4-11-node-observability-operator]
+==== Node Observability Operator (Technology Preview)
+{product-title} 4.11 introduces the Node Observability Operator in Technology Preview.
+
+Node Observability Operator provides the following features:
```

Comment:
Consider rewording to:

The Node Observability Operator provides the ability to: